pkg/cli/admin/upgrade/status/examples/4.16.0-single-node.output

```diff
@@ -0,0 +1,19 @@
+= Control Plane =
+Assessment:      Completed
+Target Version:  4.16.0-0.nightly-2024-07-24-232111 (a new install)
+Completion:      100%
+Duration:        25m
+Operator Status: 33 Healthy
+
+Control Plane Nodes
+NAME                          ASSESSMENT   PHASE     VERSION                              EST   MESSAGE
+ip-10-0-31-245.ec2.internal   Completed    Updated   4.16.0-0.nightly-2024-07-24-232111   -     
+
+= Worker Upgrade =
+
+WORKER POOL   ASSESSMENT   COMPLETION   STATUS
+worker        Empty                     0 Total
+
+= Update Health =
+SINCE    LEVEL   IMPACT   MESSAGE
+26m19s   Info    None     Update is proceeding well
```

Comment:
I would maybe prefer an active update (as the output showcases that the control plane and data plane are updated); however, my goal of these specific examples (compact and SNO) is to check the functionality of the command against different topologies of an OpenShift cluster and not so much on showcasing specific update scenarios (e.g., `compact-degraded`, `sno-degraded`,` compact-pdb-prohibits-update`, `sno-pdb-prohibits-update`,...). Thus, I think the outputs are fine as they are :+1:

Reply:
> I would maybe prefer an active update

Took the yamls again on new clusters where upgrade is ongoing.